src/mpi/coll/reduce/reduce.c

```diff
@@ -84,6 +84,19 @@
         If set to false, the device-level reduce function will not be
         called.
 
+    - name        : MPIR_CVAR_COLLECTIVE_DECISION_TREE
+      category    : COLLECTIVE
+      type        : boolean
+      default     : true
+      class       : device
+      verbosity   : MPI_T_VERBOSITY_USER_BASIC
+      scope       : MPI_T_SCOPE_ALL_EQ
+      description : >-
+        TODO - Move this to the correct place later.
+        If set to true, MPICH will use the collective decision tree
+        information to make a decision about which collective
+        algorithm to use. Otherwise, the older if/else model (or
+        CVARs if manually chosen) will be used.
```

Comment:
Should this be a per collective CVAR? Imagine a scenario where decision tree is default but the user wants to force a certain algorithm for a specific collective operation.

Reply:
Perhaps. For now, I'm going with something simpler, but we can always add more later if we want or cut back to use the existing CVARs.